draft-ietf-quic-version-negotiation.md

```diff
@@ -392,13 +392,32 @@ its length is not divisible by four), then the endpoint MUST close the
 connection; if the connection was using QUIC version 1, that connection closure
 MUST use a transport error of type TRANSPORT_PARAMETER_ERROR. If an endpoint
 receives a Chosen Version equal to zero, or any Available Version equal to zero,
-it MUST treat it as a parsing failure.
+it MUST treat it as a parsing failure. If a server receives a Version
```

Comment:
What is a parsing failure?  TRANSPORT_PARAMETER_ERROR or something else?

Reply:
That's described earlier in this paragraph

Reply:
No, it is not.  You really need to use the words "parsing failure" in the definition if you mean to invoke it with this text.

Reply:
Sure. Fixed by explicitly using the words "parsing failure" above.